docs/reference/requirement-sets/excel-preview-apis.md

```diff
@@ -88,11 +95,61 @@ The following table lists the Excel JavaScript APIs currently in preview. For a
 |[TableFilteredEventArgs](/javascript/api/excel/excel.tablefilteredeventargs)|[tableId](/javascript/api/excel/excel.tablefilteredeventargs#tableid)|Gets the id of the table in which the filter is applied.|
 ||[type](/javascript/api/excel/excel.tablefilteredeventargs#type)|Gets the type of the event.|
 ||[worksheetId](/javascript/api/excel/excel.tablefilteredeventargs#worksheetid)|Gets the id of the worksheet which contains the table.|
+|[Task](/javascript/api/excel/excel.task)|[addAssignee(email: string)](/javascript/api/excel/excel.task#addassignee-email-)|Adds an assignee to the task.|
+||[applyChanges(taskChanges: Excel.TaskChanges)](/javascript/api/excel/excel.task#applychanges-taskchanges-)|Applies the given changes to the task.|
+||[assignees](/javascript/api/excel/excel.task#assignees)|Gets the users to whom the task is assigned.|
+||[comment](/javascript/api/excel/excel.task#comment)|Gets the comment associated with the task.|
+||[dueDate](/javascript/api/excel/excel.task#duedate)|Gets the date and time the task is due.|
+||[historyRecords](/javascript/api/excel/excel.task#historyrecords)|Gets the history records of the task.|
+||[id](/javascript/api/excel/excel.task#id)|Gets the id of the task.|
+||[percentComplete](/javascript/api/excel/excel.task#percentcomplete)|Gets the completion percentage of the task.|
+||[priority](/javascript/api/excel/excel.task#priority)|Gets the priority of the task.|
+||[startDate](/javascript/api/excel/excel.task#startdate)|Gets the date and time the task should start.|
+||[title](/javascript/api/excel/excel.task#title)|Gets title of the task.|
+||[removeAllAssignees()](/javascript/api/excel/excel.task#removeallassignees--)|Removes all assignees from the task.|
+||[removeAssignee(email: string)](/javascript/api/excel/excel.task#removeassignee-email-)|Removes an assignee from the task.|
+||[setPercentComplete(percentComplete: number)](/javascript/api/excel/excel.task#setpercentcomplete-percentcomplete-)|Changes the completion of the task.|
+||[setPriority(priority: number)](/javascript/api/excel/excel.task#setpriority-priority-)|Changes the priority of the task.|
+||[setStartDateAndDueDate(startDate: Date, dueDate: Date)](/javascript/api/excel/excel.task#setstartdateandduedate-startdate--duedate-)|Changes the start and the due dates of the task.|
+||[setTitle(title: string)](/javascript/api/excel/excel.task#settitle-title-)|Changes the title of the task.|
+|[TaskChanges](/javascript/api/excel/excel.taskchanges)|[dueDate](/javascript/api/excel/excel.taskchanges#duedate)|Sets a new due date for the task, in UTC time zone.|
```

Comment:
Is the comma needed here?

Reply:
I think it is. It's a parenthetical note.